Markdown/Markdown.sublime-syntax

```diff
@@ -180,7 +180,21 @@ variables:
     #     \s*$      # any amount of whitespace until EOL
     #   )
 contexts:
+  file-start:
+    - match: ^---$\n
+      scope: punctuation.section.block.begin.frontmatter.yaml.markdown
```

Comment:
Do we really want to go the way to add such subscopes?

There is no corresponding meta scope for frontmatters.

Result:

```
    scope: meta.frontmatter.markdown punctuation.section.frontmatter.begin.markdown
```